monitor/grafana/zeebe.json

```diff
@@ -4589,42 +4726,34 @@
           "pointradius": 5,
           "points": false,
           "renderer": "flot",
-          "seriesOverrides": [
-            {
-              "alias": "/.*limit.*/",
-              "color": "#C4162A",
-              "fill": 0,
-              "linewidth": 2
-            }
-          ],
+          "seriesOverrides": [],
           "spaceLength": 10,
           "stack": false,
           "steppedLine": false,
           "targets": [
             {
-              "expr": "process_files_open_files{namespace=~\"$namespace\",pod=~\"$pod\"}",
-              "format": "time_series",
-              "hide": false,
-              "interval": "",
-              "intervalFactor": 1,
-              "legendFormat": "{{pod}} use fds",
+              "expr": "(kubelet_volume_stats_used_bytes{namespace=~\"$namespace\", persistentvolumeclaim=~\".*zeebe.*\"} / kubelet_volume_stats_capacity_bytes{namespace=~\"$namespace\", persistentvolumeclaim=~\".*zeebe.*\"})",
```

Comment:
:thought_bubble: I can't remember, but would it be possible to use the `$pod` variable at all here? It's a little out of scope since I know you just moved this, but you know, boy scout rule and all :smile:

Reply:
This metric has no pod label